cmd/formula-analytics.rb

```diff
@@ -305,7 +305,7 @@ def format_dimension(dimension, key)
     when "10.13" then "macOS High Sierra (10.13)"
     when "10.14" then "macOS Mojave (10.14)"
     when "10.15" then "macOS Catalina (10.15)"
-    when "10.16", "11.0" then "macOS Big Sur (#{dimension})"
+    when "10.16", /^11\./ then "macOS Big Sur (11)"
```

Comment:
Actually, looking at https://formulae.brew.sh/analytics/os-version/30d/, I think this should be `"macOS Big Sur (#{dimension})"` after all. Opened https://github.com/Homebrew/homebrew-formula-analytics/pull/191